src/clockkit.cs

```diff
@@ -1,4 +1,4 @@
-﻿//
```

Comment:
UTF-8 BOM? Thank you text editor...

Reply:
Other way around, my editor does not like BOM 💣